server/src/internalClusterTest/java/org/elasticsearch/cluster/settings/ClusterSettingsIT.java

```diff
@@ -51,11 +50,11 @@ public void testClusterNonExistingSettingsUpdate() {
         try {
             client().admin().cluster()
                     .prepareUpdateSettings()
-                    .setTransientSettings(Settings.builder().put(key1, value1).build())
```

Comment:
This is another test case where we may want to keep some dedicated tested for transient settings.